test/verify.cc

```diff
@@ -6,6 +6,8 @@
 #include <stdio.h>   // printf
 #include <string.h>  // memcpy
 
+#include <bitset>
```

Comment:
This looks unused.

Reply:
Yes, a stray one, I wanted  to reinvent `bit_floor` in C++17, but decided that using an odd concurrency value as it is would be easier.